drivers/hmis/app/graphql/mutations/create_units.rb

```diff
@@ -0,0 +1,34 @@
+module Mutations
+  class CreateUnits < BaseMutation
+    argument :input, Types::HmisSchema::UnitInput, required: true
+
+    field :inventory, Types::HmisSchema::Inventory, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false
+
+    def resolve(input:)
+      inventory = Hmis::Hud::Inventory.editable_by(current_user).find_by(id: input.inventory_id)
+      return { inventory => nil, errors: [InputValidationError.new('Inventory record not found', attribute: 'id')] } unless inventory.present?
+
+      return { inventory => nil, errors: [InputValidationError.new('Unit count must be non-negative', attribute: 'count')] } unless input.count&.positive?
```

Comment:
Can this be zero?  Maybe the error message can just say that the unit count must be positive if not.  If it can be zero, then it should check that.

Reply:
It can be zero. Updated to only error if its a negative number